#Explicit Apriltag declaration
april_tags:
  [
          {id: 0,  x: 3.0000, y: 1.0000, z: 0.5, qw: 0.866, qx: 0.5, qy: 0.0, qz: 0.0},
          {id: 1,  x: 1.5582, y: 3.0, z: 0.6, qw: 0.965, qx: 0.0, qy: -0.258819, qz: 0.0},
          {id: 2,  x: 0.0548, y: -1.0522, z: 0.8, qw: 0.866, qx: -0.5, qy: 0.0, qz: 0.0},
          {id: 3,  x: -1.580, y: -2.0553, z: 0.5, qw: 0.965, qx: 0.0, qy: -0.258819, qz: 0.0},
          {id: 4,  x: -2.0543, y: 1.0603, z: 0.7, qw: 1.0, qx: 0.0, qy: 0.0, qz: 0.0}
  ]
#Only used in Randomized
rand: False   #Enables Randomization
numTags: 5 
#Declare a Box where you want to randomly spawn Tags
#Yaxis
maxY: 3
minY: -3
#Xaxis
maxX: 3
minX: -3
#Zaxis
maxZ: 3
minZ: 1
